### PR TITLE
[Merged by Bors] - feat: add lemmas about `Inv.inv` for `Filter`s

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Order/Filter/AtTopBot.lean [hunk 1]
```diff
@@ -884,12 +884,10 @@ theorem map_neg_atTop : map (Neg.neg : β → β) atTop = atBot :=
   (OrderIso.neg β).map_atTop
 #align filter.map_neg_at_top Filter.map_neg_atTop
 
-@[simp]
 theorem comap_neg_atBot : comap (Neg.neg : β → β) atBot = atTop :=
```
Comment (eric-wieser) [inline Mathlib/Order/Filter/AtTopBot.lean:887]: Should the statement be changed here to `-atBot = atTop`? Then it could still be `simp`.

Comment (eric-wieser) [reply Mathlib/Order/Filter/AtTopBot.lean:887]: Oh nevermind, I guess that's the lemma in the other file.

This whole section looks like it should be generalized to `OrderedCommGroup`...

File: Mathlib/Topology/Algebra/GroupWithZero.lean [hunk 2]
```diff
@@ -146,6 +146,20 @@ theorem Units.embedding_val₀ [GroupWithZero G₀] [TopologicalSpace G₀] [Has
...
 
+section NhdsInv
+
+variable [GroupWithZero G₀] [TopologicalSpace G₀] [HasContinuousInv₀ G₀] {x : G₀}
+
+lemma nhds_inv₀ (hx : x ≠ 0) : 𝓝 x⁻¹ = (𝓝 x)⁻¹ := by
+  refine le_antisymm (inv_le_iff_le_inv.1 ?_) (tendsto_inv₀ hx)
+  simpa only [inv_inv] using tendsto_inv₀ (inv_ne_zero hx)
+
+lemma tendsto_inv_iff₀ {α : Type*} {l : Filter α} {f : α → G₀} (hx : x ≠ 0) :
```
Comment (eric-wieser) [inline Mathlib/Topology/Algebra/GroupWithZero.lean:157]:
```suggestion
lemma tendsto_inv_iff₀ {l : Filter α} {f : α → G₀} (hx : x ≠ 0) :
```

**Review by eric-wieser — APPROVED** [review]
bors d+

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**